www/content/faq.md

```diff
@@ -0,0 +1,447 @@
+# faq
```

Comment:
Can we expand this to the full Frequently Asked Questions

Reply:
Ok, will do that now